packages/compass-e2e-tests/helpers/result-logger.js

```diff
@@ -2,7 +2,7 @@
 const assert = require('assert');
 const Mocha = require('mocha');
 
-const debug = require('debug')('compass-e2e-tests').extend('result-logger');
```

Comment:
Why did the prefix got removed here?

Reply:
I explained in the description:

> ResultLogger now has its own debug pattern rather than extending compass-e2e-tests which is handy because we can now more easily ignore its rather verbose and usually useless output

I considered just removing debug from the module since it is noisy and rarely useful unless you're working on result logger, but that felt too much.

Reply:
Totally glanced over the drive-by section without paying too much attention, thanks for context!